meinberlin/apps/votes/assets/token_input.js

```diff
@@ -0,0 +1,109 @@
+function init () {
```

Comment:
@goapunk @khamui so I got this from a stackoverflow suggestion on how to autoformat a credit card entry, I have kind of cleaned it but not enough but before i spend more time I wanted to sanity check with you guys?